x-pack/plugins/advanced_ui_actions/public/components/action_wizard/test_data.tsx

```diff
@@ -0,0 +1,150 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useState } from 'react';
+import { EuiFieldText, EuiFormRow, EuiSelect, EuiSwitch } from '@elastic/eui';
+import { ActionFactory, ActionFactoryBaseConfig } from './action_wizard';
+
+export const dashboards = [
+  { id: 'dashboard1', title: 'Dashboard 1' },
+  { id: 'dashboard2', title: 'Dashboard 2' },
+];
+
+export const DashboardDrilldownActionFactory: ActionFactory<
```

Comment:
```suggestion
export const dashboardDrilldownActionFactory: ActionFactory<
```

since it's an instance/object, not a type/class.

Reply:
fixed